tests/unit/core/test_tracking.js

```diff
@@ -27,25 +31,25 @@ import {
   UNINSTALL_THEME_CATEGORY,
 } from 'core/constants';
 
+function stubConfig(overrides = {}) {
+  const config = {
+    trackingEnabled: true,
+    trackingId: 'sample-tracking-id',
+    ...overrides,
+  };
+  return { get: sinon.spy((key) => config[key]) };
+}
+
+function createTracking(overrides = {}) {
```

Comment:
We also have a `createFakeTracking()` helper in `tests/unit/helpers` if you don't need the full object.

Reply:
Given the test cases below, I think you need the full `Tracking` object so forget this comment.